sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Azure.Messaging.EventHubs.Processor.csproj

```diff
@@ -1,8 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Azure Event Hubs is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers.  This library extends its Event Processor with durable storage for checkpoint information using Azure Blob storage.  For more information about Event Hubs, see https://azure.microsoft.com/en-us/services/event-hubs/</Description>
-    <Version>5.3.1</Version>
-    <ApiCompatVersion>5.3.0</ApiCompatVersion>
+    <Version>5.4.0-beta.1</Version>
+    <ApiCompatVersion>5.3.1</ApiCompatVersion>
```

Comment:
This shouldn't be manually updated.

Reply:
This is intentional.  It's being manually updated due to the base branch that does not contain the post-release update to the latest GA API  (v5.3.1)